### PR TITLE
Experiment with #213

### PR DESCRIPTION
**Pull request by lgoettgens**

### CODE REVIEW

File: .github/workflows/code-style.yml [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
 jobs:
-  code-style:
+  format:
     runs-on: ubuntu-latest
     steps:
-      - uses: tkf/julia-code-style-suggesters@v1
+      - uses: actions/checkout@v2
```
Comment (fingolfin) [inline .github/workflows/code-style.yml:16]:
```suggestion
      - uses: actions/checkout@v4
```

File: .github/workflows/code-style.yml [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+      - uses: julia-actions/setup-julia@latest
```
Comment (fingolfin) [inline .github/workflows/code-style.yml:17]:
```suggestion
      - uses: julia-actions/setup-julia@v1
```

File: .github/workflows/code-style.yml [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+        with:
+          version: 1
```
Comment (fingolfin) [inline .github/workflows/code-style.yml:19]: Version 1 is the default anyway.
```suggestion
```

File: .github/workflows/code-style.yml [hunk 4]
```diff
@@ -1,10 +1,27 @@
...
+      - run: |
+          julia  -e 'using Pkg; Pkg.add("JuliaFormatter")'
+          julia  -e 'using JuliaFormatter; format("."; verbose=true)'
```
Comment (fingolfin) [inline .github/workflows/code-style.yml:22]:
```suggestion
          julia --color=yes -e 'using Pkg; Pkg.add("JuliaFormatter")'
          julia --color=yes -e 'using JuliaFormatter; format("."; verbose=true)'
```

File: .github/workflows/code-style.yml [hunk 5]
```diff
@@ -1,10 +1,27 @@
...
+      - uses: reviewdog/action-suggester@v1
+        with:
+          tool_name: JuliaFormatter
```
Comment (fingolfin) [inline .github/workflows/code-style.yml:25]: I've followed the instructions under https://github.com/reviewdog/reviewdog#option-2-install-reviewdog-github-apps
So let's try this:
```suggestion
          github_token: ${{ secrets.REVIEWDOG_TOKEN }}
          tool_name: JuliaFormatter
```

**Comment by lgoettgens** [comment]
These suggestions should go to https://github.com/JuliaTesting/Aqua.jl/pull/213. This PR is only to test that everything works.

**Comment by lgoettgens** [comment]
Rebased this on top of the changes (@fingolfin's suggestions) in #213.

**Comment by fingolfin** [comment]
Well, I find it quite cumbersome for experiments, I don't want to update two PRs. Maybe move the test commit in this PR to #213, close this one, and then can we can edit #213 until it works. And drop the "tests" once everything works

File: .github/workflows/code-style.yml [hunk 6]
```diff
@@ -1,10 +1,26 @@
...
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
 jobs:
-  code-style:
+  format:
     runs-on: ubuntu-latest
```
Comment (fingolfin) [inline .github/workflows/code-style.yml:14]:
```suggestion
    runs-on: ubuntu-latest
    env:
      REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.REVIEWDOG_TOKEN }}
```

File: .github/workflows/code-style.yml [hunk 7]
```diff
@@ -1,10 +1,28 @@
...
-      - uses: tkf/julia-code-style-suggesters@v1
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v1
+      - run: |
+          julia --color=yes -e 'using Pkg; Pkg.add("JuliaFormatter")'
+          julia --color=yes -e 'using JuliaFormatter; format("."; verbose=true)'
+      - uses: reviewdog/action-suggester@v1
+        with:
+          tool_name: JuliaFormatter
+          github_token: ${{ secrets.REVIEWDOG_TOKEN }}
```
Comment (fingolfin) [inline .github/workflows/code-style.yml:26]:
```suggestion
          github_token:  ${{ secrets.github_token }}
```

**Pull request closed without merging**